common/src/main/resources/figura-common.mixins.json

```diff
@@ -4,6 +4,7 @@
   "package": "org.figuramc.figura.mixin",
   "compatibilityLevel": "JAVA_17",
   "mixins": [
+    "render.renderers.TridentRendererMixin"
```

Comment:
Please remove this, we don't register our mixins on the common side, only on the client, you have it here and down with the client stuff as well, with this removed the rest looks good to me, so it could still be merged in time for 0.1.3